shop/src/main/java/ro/msg/learning/shop/ShopApplication.java

```diff
@@ -0,0 +1,19 @@
+package ro.msg.learning.shop;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+public class ShopApplication {
+    @Bean
+    public ModelMapper modelMapper(){
```

Comment:
You should define a specific configuration for your ModelMapper. The main SpringBootApplication class should stay the same, maybe with some extra annotations if necessary.